### PR TITLE
Add documentation for undocumented methods in Levenshtein module

### PR DESCRIPTION
**Pull request by katafrakt**
Adds documentation for `Levensthein.find` (both variants).

### CODE REVIEW

File: src/levenshtein.cr [hunk 1]
```diff
@@ -101,12 +101,34 @@ module Levenshtein
     end
   end
 
+  # Finds the best match for *name* among strings added within a block.
+  # *Tolerance* can be used to set maximum Levenshtein distance allowed.
```
Comment (straight-shoota) [inline src/levenshtein.cr]:
```suggestion
  # *tolerance* can be used to set maximum Levenshtein distance allowed.
```

File: src/levenshtein.cr [hunk 2]
```diff
@@ -101,12 +101,34 @@ module Levenshtein
     end
   end
 
+  # Finds the best match for *name* among strings added within a block.
```
Comment (straight-shoota) [inline src/levenshtein.cr]:
```suggestion
  # Finds the best match for *name* among strings added within the given block.
```

File: src/levenshtein.cr [hunk 3]
```diff
@@ -101,12 +101,34 @@ module Levenshtein
...
+  # end # => "ello"
+  # ```
   def self.find(name, tolerance = nil)
     Finder.find(name, tolerance) do |sn|
       yield sn
     end
   end
 
+  # Finds the best match for *name* among strings provided in *all_names*.
+  # *Tolerance* can be used to set maximum Levenshtein distance allowed.
```
Comment (straight-shoota) [inline src/levenshtein.cr]:
```suggestion
  # *tolerance* can be used to set maximum Levenshtein distance allowed.
```

**Comment by katafrakt** [comment]
@straight-shoota Thanks for the suggestions. I updated the PR.

**Comment by asterite** [comment]
I'd actually like to remove `Levenshtein` from the std, or at least rewrite the current API. It was mostly written for the compiler. Same thing that happened with the `Markdown` module.

**Comment by straight-shoota** [comment]
@asterite It still doesn't hurt to document it.

**Comment by asterite** [comment]
I'd merge this but...

> The base branch restricts merging to authorized users

🤔

**Comment by katafrakt** [comment]
@bcardiff thanks, fixed.

**Comment by straight-shoota** [comment]
Dziękuję @katafrakt

**Pull request merged**